packages/aws-appsync/src/client.ts

```diff
@@ -260,7 +260,7 @@ class AWSAppSyncClient<TCacheShape extends NormalizedCacheObject> extends Apollo
         return !this._disableOffline;
     }
 
-    mutate<T, TVariables = OperationVariables>(options: MutationOptions<T, TVariables>): Promise<FetchResult<T>> {
+    mutate<T, TVariables = OperationVariables>(options: MutationOptions<T, TVariables>) {
```

Comment:
Why the removal of the Promise? Is this a breaking change?

Reply:
since we are just returning `super.mutate(...)`, TypeScript can infer the return type automatically, that's why I removed it